README.md

```diff
@@ -363,6 +364,89 @@ However, in the following scenario the default locale will be used, although it
+| -------- | ---- | -------- | ----------- |
+| urlPattern | `string` | No | A pattern string (which will be converted a regular expression) that matches a specific route. |
+| strategy | `string` | No | The caching strategy (see below). |
+| options | `object` | Yes | An object of additional options, each detailed below. |
+| cacheName | `string` | Yes | The name of the cache to use for the route. Note that the `cachePrefix` is _not_ prepended to the cache name. Defaults to the main runtime cache (`${cachePrefix}-runtime-${domain}`). |
+| cacheableResponse | `object` | Yes | Uses HTTP status codes and or headers to determine whether a response can be cached. This object has two optional properties: `statuses` and `headers`. `statuses` is an array of HTTP status codes that should be considered valid for the cache. `headers` is an object of HTTP header and value pairs; at least one header must match for the response to be considered valid. Defaults to `{ statuses: [ 200 ] }` when the `strategy` is `'cacheFirst'`, and `{ statuses: [0, 200] }` when the `strategy` is either `networkFirst` or `staleWhileRevalidate`. |
+| expiration | `object` | Yes | Controls how the cache is invalidated. This object has two optional properties. `maxEntries` is the number of responses that can be cached at any given time. Once this max is exceeded, the oldest entry is removed. `maxAgeSeconds` is oldest a cached response can be in seconds before it is removed. |
+| networkTimeoutSeconds | `number` | Yes | Used with the `networkFirst` strategy to specify how long in seconds to wait for a resource to load before falling back on the cache. |
+
+
+#### Strategies
+
+Four routing strategies are currently supported: `networkFirst`, `cacheFirst`, `networkOnly`, and `staleWhileRevalidate`.
+
+`networkFirst` attempts to load a resource over the network, falling back on the cache if the request fails or times out. This is a useful strategy for assets that either change frequently or may change frequently (i.e., are not versioned). With the `cacheFirst` strategy, a resource is loaded from the cache unless it does not exist, in which case it is fetched over the network. `cacheFirst` is best for resources that change in frequently or can be cached for a long time (e.g., versioned assets). The `networkOnly` strategy forces the resource to always be retrieved over the network, and is useful for requests that have no offline equivalent. Finally, `staleWhileRevalidate` requests resources from both the cache and the network simulaneously. The cache is updated with each successful network response. This strategy is best for resources that do not need to be continuously up-to-date, like user avatars. However, when fetching third-party resources that do not send CORS headers, it is not possible to read the contents of the response or verify the status code. As such, it is possible that a bad response could be cached. In such cases, the `networkFirst` strategy may be a better fit.
```

Comment:
"is best for resources that change in frequently or can be cached for a long time" -> did you mean infrequently?